lib/chai/core/assertions.js

```diff
@@ -751,25 +1109,44 @@ module.exports = function (chai, _) {
   Assertion.addMethod('greaterThan', assertAbove);
 
   /**
-   * ### .least(value)
+   * ### .least(n[, msg])
+   *
+   * Asserts that the target is a number greater than or equal to the given
+   * number `n`. However, it's often best to assert that the target is equal to
```

Comment:
IMO it makes thinks strange to have a feature and then discourage users to use it unless we show them a good reason to do it.
Maybe we could add a simple explanation telling them that when they assert `something` is not `<other_thing>` this means that `something` can be `Inifinity - 1` things, which is too loose.

What do you think? I'm not sure this would be too much or not.